src/workspacer.Native/Native/WindowsManager.cs

```diff
@@ -89,6 +89,19 @@ public void DumpWindowDebugOutput()
             OpenDebugOutput(output);
         }
 
+        public void DumpWindowDebugOutputForFocusedWindow()
+        {
+            var output = "";
+            var focusedWindow = Windows.Where(win => win.IsFocused)
+                .Select(win => win).FirstOrDefault();
+
+            if (focusedWindow != null)
+            {
+                output += GenerateWindowDebugOutput(focusedWindow) + "\r\n\r\n";
```

Comment:
Output has previously only been assigned a blank value, and we don't need to concatenate to it. `output = ` alone will be fine.

Also: Any reason we need to append those newlines at the end? :smile:

Reply:
Done